app/views/vm_common/_live_migrate.html.haml

```diff
@@ -0,0 +1,66 @@
+%form#form_div{:name => "angularForm", 'ng-controller' => "vmCloudLiveMigrateFormController"}
+  = render :partial => "layouts/flash_msg"
+  %h3
+    = _('Migrate Instance')
+  .form-horizontal
+    .form-group
+      %label.col-md-2.control-label
+        = _('Auto-select Host?')
+      .col-md-8
+        = check_box_tag("auto_select_host",
+                        '1',
+                        @edit[:auto_select_host],
+                        'ng-model'     => "vmCloudModel.auto_select_host",
+                        :miqrequired   => true,
+                        :checkchange   => true)
```

Comment:
@mansam For rendering checkboxes in an angular form, use `%input`
Something like this -
```
%input{:type            => "checkbox",
       :name            => "auto_select_host",
       'ng-model'       => "vmCloudModel.auto_select_host",
       :checkchange     => true}
```
We definitely don't need the `@edit`